keyboards/cx60/config.h

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2019 Rodrigo Feijao <hello@rodrigofeijao.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x4358 // "CX"
+#define PRODUCT_ID      0x3630 // "60"
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    CX60
+#define PRODUCT         CX60-VIA
+#define DESCRIPTION     CX60, VIA-Enabled
+
+/* key matrix size */
+#define MATRIX_ROWS 5
+#define MATRIX_COLS 14
+
+/* Keyboard Matrix Assignments */
+#define MATRIX_ROW_PINS { F1, F4, F5, F6, E6 }
+#define MATRIX_COL_PINS { C7, C6, F7, F0, B4, D7, D6, B0, B1, B2, B3, D2, D3, D5 }
+#define UNUSED_PINS
+
+/* Backlight Setup */
+#define BACKLIGHT_PIN B7
+#ifdef BACKLIGHT_PIN
+#define BACKLIGHT_LEVELS 5
+#endif
+
+/* COL2ROW or ROW2COL */
+#define DIODE_DIRECTION COL2ROW
+
+/* Debounce reduces chatter (unintended double-presses) - set 0 if debouncing is not needed */
+#define DEBOUNCE 5
+
+/* RGB Underglow */
+#define RGB_DI_PIN B6
+#define RGBLIGHT_ANIMATIONS
+#define RGBLED_NUM 14
+#define RGBLIGHT_HUE_STEP 10
+#define RGBLIGHT_SAT_STEP 17
+#define RGBLIGHT_VAL_STEP 17
+
```

Comment:
per https://docs.qmk.fm/#/feature_led_indicators?id=configuration-options

```suggestion

#define LED_CAPS_LOCK_PIN B5
```

Reply:
if we used the codes, capslock led behaves normally. if we removed it and replaced with #define LED_CAPS_LOCK_PIN, capslock led behaves in the opposite way

Reply:
`bool led_update_kb(led_t led_state) {
    bool res = led_update_user(led_state);
    if(res) {
        writePin(B5, !led_state.caps_lock);
    }
    return res;
}`

Can we add this code in cx60.c? If this code is added, it will work. Thanks

Reply:
You may need: 
```
#define LED_PIN_ON_STATE 0
```

Otherwise, yes, absolutely, you can. 
https://docs.qmk.fm/#/feature_led_indicators?id=example-led_update_kb-implementation

The defines are for "simple" configurations that don't need anything special.  Ideally, these should be used. But that isn't always possible.

Reply:
Thanks.. its work without added code..

Reply:
Glad to hear it!